Multisig.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -181,6 +217,15 @@
           "version": null
         }
       },
+      {
+        "package": "SafeWeb3",
+        "repositoryURL": "https://github.com/safe-global/SafeWeb3.swift",
+        "state": {
+          "branch": "rename-to-SafeWeb3",
```

Comment:
can we set branch to null here and make SafeWeb3 available by default?

Reply:
No, I need that branch specifically, however this file is auto generated